mapper/tedge_mapper/src/c8y_mapper.rs

```diff
@@ -1,29 +1,44 @@
+use crate::c8y_converter::CumulocityConverter;
+use crate::component::TEdgeComponent;
 use crate::mapper::*;
-use mqtt_client::Topic;
+use async_trait::async_trait;
+use tedge_config::TEdgeConfig;
+use tracing::{debug_span, Instrument};
 
-#[derive(Debug)]
-pub struct CumulocityMapperConfig {
-    pub in_topic: Topic,
-    pub out_topic: Topic,
-    pub errors_topic: Topic,
-}
+const CUMULOCITY_MAPPER_NAME: &str = "tedge-mapper-c8y";
+
+pub struct CumulocityMapper {}
```

Comment:
Can we please add `#[derive(Debug)]` to the struct? It makes easier for any kind of debugging and it is recommended practice.
```suggestion
#[derive(Debug)]
pub struct CumulocityMapper {}
```

Reply:
I'll do that.